### ISSUE TITLE
Lack of Accessibility

### ISSUE THREAD

**Issue by JaffarSidekJaffar** [czyzby/gdx-setup#59]
### Hi.  I found your Alternative Gdx.jar while looking up for some skins that Me and my friends could use for developing a game.  I am using LibGdx for developing our game and so used the mainstream gdx.jar setup app to create my project.  The mainstream version is til now, accessible, that is to say, I am able to fill up and tick the appropriate boxes to lay the foundations for my game project.  I am a blind computer software developer building a game with the help of sighted friends for graphics Guis and such, so I rely heavily on a program being accessible to my screen reader for me to perform the needed tasks.  I wonder if there is a chance for your alternative setup program to be as accessible as the mainstream one?  I am atracted to your alternative setup enhancers as well as the alternative 3rd party tools and extensions that I could potentially use.  Thanks and cheers!

**Comment by czyzby** [comment]
Hi @JaffarSidekJaffar, thanks for your input. The official gdx-setup uses Swing, which seems to have built-in accessibility. This application was created using LibGDX itself - and since LibGDX unfortunately does not support screen readers, neither do we. I'm afraid this would require changes in the framework itself - you could try adding an issue in the official LibGDX repository.

If you're mostly interested in third-party tools though, going through this file might help you:
https://github.com/czyzby/gdx-setup/blob/master/src/main/kotlin/com/github/czyzby/setup/data/libs/unofficial/thirdPartyExtensions.kt
It lists all of the extensions, their latest versions, Gradle dependencies and URLs to their websites.
